pom.xml

```diff
@@ -44,7 +44,7 @@
         <commons-logging.version>1.2</commons-logging.version>
         <consul-client.version>1.5.1</consul-client.version>
         <curator.version>5.1.0</curator.version>
-        <dropwizard.version>2.0.25</dropwizard.version>
+        <dropwizard.version>2.0.26</dropwizard.version>
         <dropwizard.metrics.version>4.1.25</dropwizard.metrics.version>
```

Comment:
doesn't DW 2.0.26 use metrics 4.1.27?

Reply:
Yes, also jersey and jetty updated.  So here is a question, should I go through the DW dependency bom and update everything to match now?  Or wait for dependabot to mark things.  I ran enforcer on all the libraries with only these changes and this is all that was flagged, but we would eventually update all the dependencies.  Thoughts?

Reply:
So your options are:

1. Run through DW's dependency BOM and update all our matching library versions
2. Just update this and then continue to push through dependabot updates
3. Do something else crazy and instead of adding each of these dependencies directly, add the DW dependency BOM as an import into our BOM.  DW does this for Jackson.

Reply:
I think either the first or third option, unless you really want to wait for all the dependabot updates and then cross-reference to make sure you're not getting the wrong version (which you most likely will since it's going to give the latest version, not necessarily the one DW 2.0.x is using)

Reply:
And Dropwizard 2.0.27 was just released, I assume because of the log4j problems...